src/main/java/uk/gov/justice/digital/delius/controller/secure/DeliusDocumentsController.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.justice.digital.delius.controller.advice.ErrorResponse;
+import uk.gov.justice.digital.delius.data.api.UploadedDocumentCreateResponse;
+import uk.gov.justice.digital.delius.service.DeliusDocumentsService;
+
+@RestController
+@RequestMapping(value = "secure", produces = MediaType.APPLICATION_JSON_VALUE)
+@AllArgsConstructor
+public class DeliusDocumentsController {
+
+    private DeliusDocumentsService deliusDocumentsService;
+    private final static String CONTACT_TYPE = "EASU";
+
+    @PostMapping(path = "/offender/{crn}/event/{eventId}/document", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    @ApiResponses(
```

Comment:
by convention the community api uses `conviction` and `convictionId` rather than `event` and `eventId` you might want to renames to be better aligned with that language

Reply:
Also, should `offender` and `event`/`conviction` be pluralised for consistency with existing endpoints?

Reply:
updated the urls for consistency to `/offenders/{crn}/convictions/{convictionId}/document`